config/config.go

```diff
@@ -148,6 +148,7 @@ type LoggingConfig struct {
 	DisableLogForwardedFor bool         `yaml:"disable_log_forwarded_for"`
 	DisableLogSourceIP     bool         `yaml:"disable_log_source_ip"`
 	RedactQueryParams      string       `yaml:"redact_query_params"`
+	AttemptsDetails        bool         `yaml:"attempts_details"`
```

Comment:
since this is a flag, I'd put this in verbal form like the other flags.
e.g.
```
LogAttemptsDetails
EnableAttemptsDetails
UseAttemptsDetails
LogDetailedTimings (see access_log.go where you did that, too)
```
etc.